frontend/src/api/files.js

```diff
@@ -85,6 +85,11 @@ export function download (format, ...files) {
 export async function post (url, content = '', overwrite = false, onupload) {
   url = removePrefix(url)
 
+  let bufferContent
+  if (content instanceof Blob && !['http:', 'https:'].includes(window.location.protocol)) {
```

Comment:
I think better would be to compare only prefix.

Reply:
that would be more performant however not strictly correct in what we are trying to catch. Someone could theoretically use a custom protocol `http+foo:` and that would cause this to break, as it is then considered a "non-http protocol" from iOS's standpoint, but would be considered an http protocol from filebrowser's.